docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1429,6 +1429,24 @@ How it **works**:
 
 - The series must already exist before exemplars can be appended, as we do not create new series upon ingesting exemplars. The series will be created when a sample from it is ingested.
 
+### err-mimir-store-consistency-check-failed
+
+This error occurs when the querier is unable to fetch some of the expected blocks after multiple retries and connections to different store-gateways. The query fails because some blocks are missing.
+
+### err-mimir-bucket-index-too-old
+
+This error occurs when a query fails because the bucket index is too old.
+
+How it **works**:
+
+- At query time, the querier and the ruler determine how old a bucket index is based on the time that it was last updated by the compactor.
+- If the age is older than the maximum stale period that is configured via `-blocks-storage.bucket-store.bucket-index.max-stale-period`, the query fails.
+- This circuit breaker ensures that the queriers and rulers do not return any partial query results due to a stale view of the long-term storage.
+
+How to **fix** it:
+
+- Increase the limit by using the `-blocks-storage.bucket-store.bucket-index.max-stale-period` option.
```

Comment:
This is not the solution. The option shouldn't be touched by users (unless they know what they're doing) so I wouldn't even mention it.

What we could say here:

```suggestion
- Ensure the compactor is running successfully (e.g. not crashing, not going out of memory).
- Ensure each compactor replica has successfully updated bucket index of each owned tenant within the double of `-compactor.cleanup-interval` (query below assumes the cleanup interval is set to 15 minutes):
  `time() - cortex_compactor_block_cleanup_last_successful_run_timestamp_seconds > 2 * (15 * 60)`
```